e2e/bitrise.yml

```diff
@@ -0,0 +1,34 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  test_test_plans:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test.git
+    - TEST_APP_BRANCH: master
+    - BITRISE_PROJECT_PATH: ./_tmp/BullsEye.xcworkspace
+    - BITRISE_SCHEME: BullsEye
+    steps:
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/env bash
```

Comment:
Just a general question: I see sometimes people use `#!/bin/env bash` for the shebang on scripts, and sometimes they use `#!/bin/bash`. Any reason to use one over the other?

Reply:
`#!/bin/env bash` [is more portable](https://stackoverflow.com/a/10383546).
and [as a rule of thumb](https://github.com/bitrise-steplib/steps-script/pull/26) `set -xeo pipefail` settings should be used.

Reply:
script steps are updated in https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/commits/1caf552f9c5fb9dda2f921436f1ab3a5d878e218